blocks/HARDWARE/OSCILLOSCOPES/TELEDYNE_LECROY/T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX/DISPLAY_ON_OFF_T3DSO1XXX.py

```diff
@@ -0,0 +1,47 @@
+from flojoy import flojoy, DataContainer, VisaConnection
+from typing import Literal, Optional
+
+
+@flojoy(inject_connection=True)
+def DISPLAY_ON_OFF_T3DSO1XXX(
+    connection: VisaConnection,
+    ch1: Literal["ON", "OFF"] = "ON",
```

Comment:
These types of parameters with 2 selections should probably be boolean, I think, from a user perspective. With True being ON for example.

Reply:
New UX:
![image](https://github.com/flojoy-ai/studio/assets/56351875/6f671ba9-2e31-42ad-b65d-66078646c5ae)